packages/utility/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "pbs-utility",
+  "name": "@plainsheet/utility",
   "version": "0.0.2",
```

Comment:
How about merging this package into just @plainsheet/core

Reply:
`@plainsheet/react` and `@plainsheet/core` share some chunks of utility functions, so I made it a separate package.